packages/core/admin/ee/admin/pages/SettingsPage/pages/ApplicationInfosPage/components/AdminSeatInfo/index.js

```diff
@@ -39,34 +40,44 @@ const AdminSeatInfo = () => {
             textColor={licenseLimitStatus === 'OVER_LIMIT' ? 'danger500' : ''}
             fontWeight={licenseLimitStatus === 'OVER_LIMIT' ? 'bold' : ''}
           >
-            {currentUserCount || 'NA'}
+            {formatMessage(
```

Comment:
@gu-stav doing it this way though messes up the colors and makes the whole message red. But according to designs only `currentUserCount` should be colored red in case it's more that `permittedSeats`. Can this be handled the way you suggested?

Reply:
Ah shit - I forgot `Typography` always sets a color. Could you check with Maeva if she would be ok to have all in red? Otherwise I can show you how you can keep it as one string even with separate components.

Reply:
Yeah, just checked with Nkem and it needs to be separated 😕

Reply:
Ok, so what you can do instead:

```js
{formatMessage({
  defaultMessage: <exceeded>{currentUserCount}</exceeded>/{permittedSeats}
}, {
  exceeded: (children) => <Typography fontWeight="semiBold" textColor="danger500" fontWeight="bold">{children}</Typography>,
})}